src/config/config.ts

```diff
@@ -104,20 +105,6 @@ export interface ConfigPropertyMetaInput {
 }
 
 export enum SfdxPropertyKeys {
-  /**
```

Comment:
We need to keep these for the foreseeable future so that sfdx-core knows how to map between the username configs used by sfdx and the ones used by sf